src/serializer/Referentializer.js

```diff
@@ -14,7 +14,7 @@ import type { SerializerOptions } from "../options.js";
 import * as t from "@babel/types";
 import generate from "@babel/generator";
 import type { BabelNodeStatement, BabelNodeExpression, BabelNodeIdentifier } from "@babel/types";
-import { NameGenerator } from "../utils/generator.js";
+import { NameGenerator } from "../utils/PreludeGenerator";
```

Comment:
Why not put NameGenerator in its own file? Importing it from generator.js seems plausible, but importing it from PreludeGenerator.js just seems weird.